plugins/gpio/GPIOPort.cpp

```diff
@@ -44,16 +45,10 @@ bool GPIOOutputPort::Init() {
 }
 
 string GPIOOutputPort::Description() const {
-  vector<uint8_t> pins = m_driver->PinList();
+  vector<uint16_t> pins = m_driver->PinList();
   std::ostringstream str;
   str << "Pins ";
-
-  vector<uint8_t>::const_iterator iter = pins.begin();
-  while (iter != pins.end()) {
-    str << static_cast<unsigned int>(*iter++);
-    if (iter != pins.end())
-      str << ", ";
-  }
+  str << ola::StringJoin(", ", pins);
   return str.str();
```

Comment:
This whole bit could now just be rewritten as:
`return "Pins " << ola::StringJoin(", ", pins);`

Then you can drop the std::ostringstream usage too.